hashicorp/waypoint
# ui: added sidebar/panel ui to deployments tab

## Screenshot
<img width="1436" alt="Screen Shot 2021-11-23 at 3 31 37 PM" src="https://user-images.githubusercontent.com/60155296/143100256-9498f44a-96be-4114-9ee6-bfddd844a7ef.png">

## Notes
- Build + release tabs are actually going to be in a different table format, so this PR just changes the deployments tab
- Changes to paths
  - The sidebar view that outlets to the details page is now the `deployment` route/template
  - The details page is the `deployment/deployment-seq` route/template
  - `deployments` route exists just to redirect to the `deployment` route
  - There's obviously some changes, but a lot of the code was just relocated
  - `StatusReportMetaTable` alignment is fixed (but this could have been from a previous PR and I just didn't notice)

## How to test
1. Pull down this branch `git checkout ui/sidebar-panel`
2. `yarn` (new `ember-route-helpers` dependency)
3. `yarn start`
4. Navigate to the deployments tab (should automatically load when you select an application)
5. Make sure everything is working as expected
